### PR TITLE
fix: add installation with prebuilt binaries

### PR DESCRIPTION
**Pull request by xilosada**

### CODE REVIEW

**Comment by xilosada** [comment]
we should separate the extra steps that are done in the build form source. They are common to everyone

File: docs/02-getting-started/01-setup.mdx [hunk 1]
```diff
@@ -3,15 +3,114 @@ id: setup
...
+
+### **Option 1: Installation Script**
+
+The installation script supports multiple terminal environments and will automatically configure your PATH based on your shell.
+
+#### Steps
+
+1. Open your terminal and run:
+   ```bash
+   curl -sSf https://raw.githubusercontent.com/calimero-network/core/master/scripts/install.sh | bash
```
Comment (xilosada) [inline docs/02-getting-started/01-setup.mdx:38]: @MatejVukosav which binaries?

Comment (MatejVukosav) [reply docs/02-getting-started/01-setup.mdx:38]: From the release section?

**Review by MatejVukosav — APPROVED** [review]

File: docs/02-getting-started/01-setup.mdx (hunk 1 above)
Comment (xilosada) [reply docs/02-getting-started/01-setup.mdx:38]: yes, but which version?

**Comment by miraclx** [comment]
1. We need to update the script, since the binary release format changed.

2. We should make it dynamic so it always pulls the latest release of both binaries.

**Pull request merged**